test/common/test_api_utils.py

```diff
@@ -1,61 +1,60 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
 
-"""
- RackHD 1.1 common test utlitiies
+RackHD common test utlitiies
 
- authors: ehohenstein, yuchi zhang
+Authors:
+Ericka Hohenstein
+yuchi zhang
+George Paulos
 
- Purpose/intent
- This utility library contain helper functions for parsing response data from the RackHD APIs.
-"""
+Purpose/intent
+This utility library contain helper functions for parsing response data from the RackHD APIs.
+'''
 
 import fit_path  # NOQA: unused import
-import sys
-import subprocess
 import fit_common
-
 import unicodedata
-import json
 
 ##########################################
 #  Node List Utils
 ##########################################
 
+
 def rackhd_get_node_list():
     """
     this function returns the list of node ids reported by
-    the Monrail API /api/1.1/nodes
+    the Monrail API /api/2.0/nodes
     :return:
         computer node list in that stack when success.
         otherwise on failure
     """
-    monurl = "/api/1.1/nodes"
+    monurl = "/api/2.0/nodes"
     mondata = fit_common.rackhdapi(url_cmd=monurl)
     nodes = mondata['json']
-    result = mondata['status']
 
     nodelist = []
     if len(nodes) == 0:
         if fit_common.VERBOSITY >= 2:
             print "No Nodes found on RackHD server "
     else:
-        inode=0
-        while inode<len(nodes):
+        inode = 0
+        while inode < len(nodes):
             nodelist.append(nodes[inode]["id"])
-            inode +=1
+            inode += 1
     return nodelist
 
 
 def get_rackhd_nodetype(nodeid):
-    nodetype = ""
+    nodetype = "unknown"
     sku = ""
     # get node info
-    mondata = fit_common.rackhdapi("/api/1.1/nodes/"+nodeid)
+    mondata = fit_common.rackhdapi("/api/2.0/nodes/" + nodeid)
     if mondata['status'] == 200:
         # get the sku id contained in the node
         sku = mondata['json'].get("sku")
         if sku:
-            skudata = fit_common.rackhdapi("/api/1.1/skus/"+sku)
+            skudata = fit_common.rackhdapi(sku)
```

Comment:
The line 56 sets the sku to the full link which includes the api/2.0 in it.
"sku": "/api/2.0/skus/621828d7-1b7e-4d06-a224-326f4680ac9e